jit/v2/JIT_v2.0.0.yaml

```diff
@@ -1701,54 +2159,341 @@ components:
           example: ALGS
         facilityTypeCode:
           type: string
-          maxLength: 4
+          maxLength: 4
+          description: |
+            A specialized version of the facilityCode to be used in the **Port Call Service**. The code to identify the specific type of facility.
+            - `PBPL` (Pilot boarding place)
+            - `BRTH` (Berth)
+            - `ANCH` (Anchorage Location)
+
+            **Condition:** `PBPL`, when used, should be used with the following `portCallServiceType`:
+              - `PILOTAGE` (Pilotage)
+              - `TOWAGE` (Towage)
+              - `PILOT_BOARDING_PLACE` (Pilot Boarding Place)
+
+            **Condition:** `BRTH`, when used, should be used with the following `portCallServiceType`:
+            - `BERTH` (Berth)
+            - `CARGO_OPERATIONS` (Cargo operations)
+            - `PILOTAGE` (Pilotage)
+            - `TOWAGE` (Towage)
+            - `MOORING` (Mooring)
+            - `BUNKERING` (Bunkering)
+            - `SLUDGE` (Sludge)
+            - `ALL_FAST` (All Fast)
+            - `GANGWAY_DOWN_AND_SECURE` (Gangway down and secure)
+            - `VESSEL_READY_FOR_CARGO_OPERATIONS` (Vessel Ready for cargo operations)
+            - `VESSEL_READY_TO_SAIL` (Vessel Ready to sail)
+            - `DISCHARGE_CARGO_OPERATIONS` (Discharge cargo operations)
+            - `LOADING_CARGO_OPERATIONS` (Loading cargo operations)
+            - `LASHING` (Lashing)
+            - `SAFETY` (Safety)
+            - `ANCHORAGE_OPERATIONS` (Anchorage Operations)
+            - `SHORE_POWER` (ShorePower)
+
+            **Condition:** `ANCH`, when used, should be used with the following `portCallServiceType`:
+            - `ANCHORAGE` (Anchorage)
+            - `ANCHORAGE_OPERATIONS` (Anchorage Operations)
+          enum:
```

Comment:
Are these fixed values, should this be an enum or pseudo enum?

Reply:
@emiliodcsa says only these three values in this version (any updates will require a new major release...)